style.css

```diff
@@ -0,0 +1,137 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,600;1,200;1,600&display=swap');
+
+.wrapper {
+  background-image: url(./images/headerbg.svg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: center center;
+  width: 100%;
+  font-family: "Poppins", sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+aside a {
+  margin-left: 30px;
+  font-weight: 500;
+  font-size: 15px;
+  color: #344563;
+}
+
+.logo a {
+  height: 20px;
+  width: 76px;
+  top: 10px;
+  left: 12px;
+  font-weight: 700;
+  color: #6070ff;
+  font-size: 18px;
+}
+
+aside {
+  margin-left: 30px;
+}
+
+.hamburger {
+  display: none;
+}
+
+.container {
+  position: absolute;
+  top: 30%;
+  left: 15%;
+  right: 15%;
+  align-items: center;
+  margin: auto;
+}
+
+.title {
+  font-size: 42px;
+  font-weight: 700;
+  height: 122px;
+  width: 920px;
+}
+
+.description {
+  height: 98px;
+  width: 920px;
+  top: 134px;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  text-align: left;
+  position: relative;
+}
+
+.socials h6 {
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: left;
+  color: #7f8cff;
+  margin-bottom: 0%;
+}
+
+.socials {
+  position: static;
+  width: 150px;
+  height: 24px;
+}
+
+@media only screen and (max-width: 600px) {
+  .wrapper {
+    background-image: url(mobilebg.svg);
```

Comment:
- [ ] [REQUIRED] As the previous code reviewer suggested.Kindly ensure the background image is properly displayed as it is in the original design.

| Your Portfolio | Template Design |
| ------------- | ------------- |
| ![Screenshot from 2022-04-21 04-57-18](https://user-images.githubusercontent.com/52934128/164369467-ebb86ae7-1029-4103-88ea-d62e253a2ccd.png)  | ![Screenshot from 2022-04-21 05-00-43](https://user-images.githubusercontent.com/52934128/164369538-8711c894-18dc-475a-bf39-dc8034f9b683.png) |